internal/services/containerapps/helpers/container_apps.go

```diff
@@ -221,7 +221,6 @@ func ContainerAppIngressCustomDomainSchema() *pluginsdk.Schema {
 	return &pluginsdk.Schema{
 		Type:     pluginsdk.TypeList,
 		Optional: true,
-		MaxItems: 1,
```

Comment:
Based on the documentation, instead of removing the line, it looks like it might have been a typo.  Should this be `MinItems : 1,` based on the documentation stating:

> One or more custom_domain block as detailed below.